appstoreconnect/api.py

```diff
@@ -669,9 +669,9 @@ def download_sales_and_trends_reports(self, filters=None, save_to=None):
 		# setup required filters if not provided
 		default_versions = {
 			'SALES': '1_0',
-			'SUBSCRIPTION': '1_2',
```

Comment:
Are these strictly part of the changes needed to support JWT >=2 or maybe part of some other change?

Reply:
part of some other change, appstore document
<img width="675" alt="Screenshot 2023-06-05 at 10 56 05" src="https://github.com/Ponytech/appstoreconnectapi/assets/132546601/dfc2e349-9eac-41b4-ae46-a7521e0b8faa">